service/storage_proxy.cc

```diff
@@ -4136,6 +4136,50 @@ storage_proxy::query_partition_key_range_concurrent(storage_proxy::clock_type::t
                 break;
             }
 
+            // Checks if 2 range bounds are contiguous, permits false negatives
+            // Assumes that provided ranges are disjoint
+            auto are_contigious = [schema](const range_bound<dht::ring_position>& first_range_end,
```

Comment:
s/contigious/contiguous

Reply:
(and I think Avi meant adding a member function to `range_bound`, not an anonymous helper)

Reply:
Done